README.rst

```diff
@@ -472,8 +472,13 @@ If you want a scraper for a new site added
 For Devs / Contribute
 ---------------------
 
-Assuming you have ``>=python3.8`` installed, navigate to the directory where you want this project to live in and drop these lines
+#### Prerequisites:
```

Comment:
```suggestion
#### Prerequisites
```

A nitpicky detail: generally section headings shouldn't have a colon (`:`) suffix at the end.

The colon can be useful to introduce some subsequent content when it's used inline (as I did in the previous paragraph), or sometimes in situations where there aren't ways to use other formatting to indicate that the content is part of the heading -- but in the case of reStructuredText (and Markdown) section titles, it is clear from the way that they are rendered that the subsequent content is part of the section.